tests/unit/test_launchers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import
 
 import os
+import sys
```

Comment:
Why was this added here?

Reply:
`sys.platform` is used in lines 248 and 277.